### DISCUSSION TITLE
How to hide specific models from the Schema section in the docs?

### DISCUSSION THREAD

**Discussion by martin-greentrax** [fastapi/fastapi#6407] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Sample model I would like to not show up in the Schema of the docs
class FooDB(FooIn, SQLModel, table=True):
    __tablename__ = "foo"
    id: int = Field(primary_key=True)
    user_id: int = Field(foreign_key="user.id", nullable=False)
    created: datetime = Field(default_factory=datetime.now)
```

### Description

I define different model classes for the same "thing". One is the database representation and one is how the incoming data is supposed to look like. E.g. the incoming would not specify the id field.
For our 3rd party clients it is helpful to see the specs of the incoming model, but I would prefer them not to see the database model. The schema section of the docs however includes both.

I know that with defaultModelsExpandDepth I can turn this section entirely off, but my question is if I can only exclude specific models from there.
 
Similar question to:
https://github.com/tiangolo/fastapi/issues/2633

### Operating System

Linux

### FastAPI Version

0.78.0

### Python Version

3.9.12

**Comment by hellocoldworld** [comment]
I think the only way to accomplish this is by manually removing it from the generated OpenAPI like described here https://fastapi.tiangolo.com/advanced/extending-openapi/#overriding-the-defaults

**Comment by martin-greentrax** [comment]
thank you :)

**Comment by tiangolo** [comment, marked as answer]
You can probably use different models and output a simpler one that doesn't include all the fields: https://sqlmodel.tiangolo.com/tutorial/fastapi/multiple-models/

Thanks for the help @hellocoldworld! And thanks for coming back and closing the issue @martin-greentrax 🍰 

> Sorry for the long delay! 🙈  I wanted to personally address each issue and they piled up through time, but now I'm checking each one in order.
